.changeset/serious-chairs-jam.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit-labs/context': patch
+---
+
+Dusting the doc
```

Comment:
This isn't a good description for a changelog.

Something like "Updated the README" is better.

Reply:
Yes sorry for the ambiguous description. I changed it.